index.md

```diff
@@ -41,6 +41,21 @@ communication. If the other party is away, and you leave before they come back,
 they can still answer your question, instead of just staring at a "Hello" and
 wondering what they missed.
 
+Some people like to probe with an initial "hello" to verify whether the presence 
+status shown is true, hoping they can determine whether the recipient is near a device,  
+then will immediately follow up with a call. Don't do that either. It's rude 
```

Comment:
I would use less subjective wording here. Some people might interpret the behaviour as rude, however I would ask for a more neutral statement, like "If someone is able and willing to answer a chat message, it does not mean that they are also available for a voice or video call."

Reply:
reworded per your suggestion.